components/membership/membership-rest/src/main/kotlin/net/corda/membership/rest/v1/MGMAdminRestResource.kt

```diff
@@ -0,0 +1,52 @@
+package net.corda.membership.rest.v1
+
+import net.corda.rest.RestResource
+import net.corda.rest.annotations.HttpPOST
+import net.corda.rest.annotations.HttpRestResource
+import net.corda.rest.annotations.RestPathParameter
+
+/**
+ * The MGM Admin API consists of endpoints used to carry out administrative tasks on membership groups. A membership
+ * group is a logical grouping of a number of Corda Identities to communicate and transact with one another with a
+ * specific set of CorDapps. The API allows the MGM to perform actions such as force decline registration requests which
+ * may be displaying unexpected behaviour. This API should only be used by the MGM and under exceptional circumstances.
+ */
+@HttpRestResource(
+    name = "MGM Admin API",
+    description = "The MGM Admin API consists of endpoints used to carry out administrative tasks on membership " +
+            "groups. A membership group is a logical grouping of a number of Corda Identities to communicate and " +
+            "transact with one another with a specific set of CorDapps. The API allows you to perform actions such as" +
+            " force decline registration requests which may be displaying unexpected behaviour. This API should only " +
+            "be used under exceptional circumstances.",
+    path = "admin"
```

Comment:
```suggestion
    path = "mgmadmin"
```
I think `/admin` is too broad.

Reply:
Changed in [db77a03](https://github.com/corda/corda-runtime-os/pull/3643/commits/db77a03e4871878327168d1dd24d95f15e893ecb)